tests/cmake/tpls.cmake

```diff
@@ -0,0 +1,137 @@
+# if(PRESSIO_ENABLE_TPL_BLAS)
+#   message(">> Enabling BLAS since PRESSIO_ENABLE_TPL_BLAS=ON ==> enabling also LAPACK")
+#   set(PRESSIO_ENABLE_TPL_LAPACK ON)
+# endif()
+# if(PRESSIO_ENABLE_TPL_LAPACK)
+#   message(">> Enabling LAPACK since PRESSIO_ENABLE_TPL_LAPACK=ON ==> enabling also BLAS")
+#   set(PRESSIO_ENABLE_TPL_BLAS ON)
+# endif()
+
+if(PRESSIO_ENABLE_TPL_MPI)
+  if(PRESSIO_ENABLE_UNIT_TESTS OR PRESSIO_ENABLE_TESTS)
+    find_package(MPI REQUIRED)
+    include_directories(${MPI_CXX_INCLUDE_PATH})
+  endif()
+endif()
+
+
+if(PRESSIO_ENABLE_TPL_BLAS OR PRESSIO_ENABLE_TPL_TRILINOS)
+    # check if BLAS_ROOT is specified
+    if (NOT ${BLAS_ROOT})
+      message("")
+      message(FATAL_ERROR "BLAS_ROOT not speificed, terminating")
+      message("Make sure you set the BLAS_ROOT env var")
+    endif()
+
+    cmake_policy(SET CMP0074 NEW)
+    find_package(BLAS REQUIRED)
+    link_libraries(${BLAS_LIBRARIES})
+    message("")
+endif()
+
+if(PRESSIO_ENABLE_TPL_LAPACK)
```

Comment:
this is needed for trilinos too like you did for blas

Reply:
Trilinos is covered with `  link_libraries(${Trilinos_LIBRARIES})` in `options.cmake`.

Reply:
https://github.com/Pressio/pressio/blob/520ff06f933f6c98200114443274746c0f809233/tests/cmake/options.cmake#L57-L58